data/autoyast_containers.xml.ep

```diff
@@ -50,11 +50,11 @@
       % } elsif ($check_var->('ARCH', 'ppc64le')) {
       <append>showopts plymouth.ignore-serial-consoles console=hvc0 console=tty mitigations=auto crashkernel=201M</append>
       <boot_boot>false</boot_boot>
-      % } elsif ($check_var->('ARCH', 'x86_64')) {
+      % } else {
       <append>splash=verbose</append>
       % }
       % if ($check_var->('ARCH', 'aarch64')) {
-      <timeout config:type="integer">20</timeout>
+      <timeout config:type="integer">1</timeout>
```

Comment:
you say `Grub timeout is 1 cause we use KEEP_GRUB_TIMEOUT on the tests using this hdd.` in the commit msg but i dont see KEEP_GRUB_TIMEOUT in the vars.json??!!

Reply:
Yeah, that's true... Maybe someone forgot to add it there :D 
Not gonna do it in this PR anyways ;) 

It is explained [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/opensusebasetest.pm#L914)... but yeah, someone should add it to `variables.md`